deployment/ansible/services.yml

```diff
@@ -6,6 +6,9 @@
     - name: Update APT cache
       apt: update_cache=yes cache_valid_time=3600
 
+    - name: Install ACL utilities
```

Comment:
It would be good to leave a commit message explaining why we need to add this and the `piplining = True` option.